Makefile

```diff
@@ -49,8 +51,8 @@ down-deps: postgres-down hashicorp-down down-networks
 run-acceptance:
 	@go test -v -tags acceptance -count=1 ./tests/acceptance
 
-run-e2e:
-	@go test -v -tags e2e -count=1 ./tests/e2e
+run-e2e: deps
```

Comment:
I wouldn't add this step before e2e. e2e are independent on the start of the dependencies.

Reply:
It still needs running dependencies. How are we supposed to run them? Isn't it with `make deps`?

Reply:
running the e2e is independent of setting your environment. You can do one `make deps` but then run 10x `make run-e2e`. These 2 processes are not linked. The e2e expect that the env is already up.

Reply:
I got it, but we should improve that workflow and the documentation about it.